### ISSUE TITLE
apng图片，开启混淆后不动了

### ISSUE THREAD

**Issue by LuoGuoXin** [penfeizhou/APNG4Android#130]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by penfeizhou** [comment]
文档中有说明：

使用前请注意!

不要将APNG资源放置到drawable或者mipmap目录下! 在Android app release构建过程中, aapt工具会压缩修改APNG资源的帧信息, 会导致播放不正常. 因此请将APNG资源放置到工程内的raw或者assets目录内.

**Comment by LuoGuoXin** [comment]
谢谢，了解
